docs/INSTALL.md

```diff
@@ -36,3 +36,13 @@ To build the Docker container, run the following command:
 ```bash
 docker build -t otava .
 ```
+
+> [!NOTE]
+> The Dockerfile contains a `--mount` option that requires BuildKit [^1].
+> The BuildKit can be installed with the following commands:
+>
+> Debian and Ubuntu: `apt install -y docker-buildx`
+>
+> Fedora: `dnf install docker-buildx`
+>
+> [^1]: https://docs.docker.com/go/buildkit/
```

Comment:
Aplogies, I'm following a bit from a distance these past weeks... But wasn't this docker issue resolved by using python ?

Reply:
`otava`'s installation guide suggest two ways: installation via pipx and installation to the docker container. It is two independent ways of installation. The second way is failed without BuildKit.

Reply:
Yeah ok, in `tox.ini` we also explicitly use `docker buildx`